### PR TITLE
update crate docker image to 0.54.11

### PR DESCRIPTION
**Pull request by kovrus**

### CODE REVIEW

File: library/crate [hunk 1]
```diff
@@ -8,8 +8,8 @@
...
 # see also https://crate.io
 
-latest: git://github.com/crate/docker-crate@1d7d393fd77f1ca795a1c4a489de5a82d53d8b85
+latest: git://github.com/crate/docker-crate@73294e9620a6194acecda8fd6e80d91653e10ade
 0.52: git://github.com/crate/docker-crate@cce8f796ba8936250eb380235cde47be494d1e95
 0.52.4: git://github.com/crate/docker-crate@cce8f796ba8936250eb380235cde47be494d1e95
-0.54: git://github.com/crate/docker-crate@1d7d393fd77f1ca795a1c4a489de5a82d53d8b85
-0.54.9: git://github.com/crate/docker-crate@1d7d393fd77f1ca795a1c4a489de5a82d53d8b85
+0.54: git://github.com/crate/docker-crate@73294e9620a6194acecda8fd6e80d91653e10ade
+0.54.9: git://github.com/crate/docker-crate@73294e9620a6194acecda8fd6e80d91653e10ade
```
Comment (tianon) [inline library/crate]: Shouldn't this tag's name be updated to `0.54.11`?

Comment (kovrus) [inline library/crate]: @tianon exactly, thanks!

**Comment by tianon** [comment]
LGTM :+1:

```diff
diff --git a/crate_latest/Dockerfile b/crate_latest/Dockerfile
index ab13aa7..580df98 100644
--- a/crate_latest/Dockerfile
+++ b/crate_latest/Dockerfile
@@ -55,7 +55,7 @@ RUN set -ex \
     && rm -rf $BUILD_DIR
 
 # install crate
-ENV CRATE_VERSION 0.54.9
+ENV CRATE_VERSION 0.54.11
 RUN apk add --no-cache --virtual .crate-rundeps openjdk8-jre-base python3 openssl \
     && apk add --no-cache --virtual .build-deps curl gnupg tar \
     && curl -fSL -O https://cdn.crate.io/downloads/releases/crate-$CRATE_VERSION.tar.gz \
```

**Comment by tianon** [comment]
Build test of #1895; 60f6a2f (`crate`):

```console

$ bashbrew build crate:latest
warning: insecure protocol git:// detected: git://github.com/crate/docker-crate
Building bashbrew/cache:ee6116519b69e5d8af0df7c926ff34a9c826e01477e4ca8bd8fa0c0028f289a4 (crate:latest)
Tagging crate:latest
Tagging crate:0.54
Tagging crate:0.54.11

$ test/run.sh crate:latest
testing crate:latest
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed


$ bashbrew build crate:0.52
Using bashbrew/cache:47e9f793dce19903b211471b66ee4a4999119b638b142e4556f18d1eef4e4f1f (crate:0.52)
Tagging crate:0.52
Tagging crate:0.52.4

$ test/run.sh crate:0.52
testing crate:0.52
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...passed
	'override-cmd' [4/4]...passed

```

**Pull request merged**